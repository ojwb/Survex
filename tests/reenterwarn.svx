; pos=no warn=1
*begin 1
*end 1
*begin 1
*end 1
*fix A 0 0 0
A B 1 0 0
