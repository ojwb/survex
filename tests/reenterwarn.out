./reenterwarn.svx:4:8: warning: Reentering an existing survey is deprecated
 *begin 1
        ^
./reenterwarn.svx:2:8: info: Originally entered here

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 2 survey stations, joined by 1 leg.
There are 0 loops.
Total length of survey legs =    1.00m (   1.00m adjusted)
Total plan length of survey legs =    1.00m
Total vertical length of survey legs =    0.00m
Vertical range = 0.00m (from b at 0.00m to b at 0.00m)
North-South range = 1.00m (from b at 1.00m to a at 0.00m)
East-West range = 0.00m (from b at 0.00m to b at 0.00m)

There were 1 warning(s).
